selftests/unit/test_utils_cpu.py

```diff
@@ -87,6 +87,229 @@ def test_s390x(self):
                             return_value=self._get_file_mock(s390x_2)):
                 self.assertEqual(len(cpu.cpu_online_list()), 4)
 
+    def test_x86_64(self):
+        x86_64 = u"""processor	: 0
+vendor_id	: GenuineIntel
```

Comment:
Please move this content to `selftests/.data`.

Reply:
I tend to favor inline content, for both locality and for making packaging easier.  It looks like you meant that moving this to a file is a must, but please clarify that to me and I'll take a look at it.

Reply:
No, not a must, just a soft-request. You can keep it here.